tune_sklearn/_trainable.py

```diff
@@ -87,8 +94,16 @@ def _train(self):
                     self.y,
                     test,
                     train_indices=train)
-                self.estimator[i].partial_fit(X_train, y_train,
-                                              np.unique(self.y))
+                if self.is_lgbm:
+                    self.saved_models[i] = self.estimator[i].fit(
+                        X_train, y_train, init_model=self.saved_models[i])
+                elif self.is_xgb:
+                    self.estimator[i].fit(
+                        X_train, y_train, xgb_model=self.saved_models[i])
+                    self.saved_models[i] = self.estimator[i].get_booster()
+                else:
+                    self.estimator[i].partial_fit(X_train, y_train,
+                                                  np.unique(self.y))
```

Comment:
what if you just put this in another trainable? would it make sense there?

Reply:
Why do we need to put it into another trainable?

Reply:
i guess it's probably going to be unsustainable to have like 5 different special cases for different libraries, but right now it's probably fine